.ci-build.yml

```diff
@@ -158,14 +124,17 @@ requires:
     - uuid-runtime
     - yelp-tools
     - xzip  # Needed to build gtkhtml from source
+    - zip
 
 variables:
   - CFLAGS="-Wall -Werror=format-security"
 
 before_scripts:
   - |
     START_DIR="$(pwd)"
+    WORK_DIR="/tmp"
```

Comment:
I think Actions has an environment variable for where it would prefer you work for scratch stuff. It might be worth reading that in here, which would also be a step towards being able to urn this same build setup locally without hard coding a path that may not be available in all build systems.

Reply:
It does, in fact. ${{ runner.temp }}. Updating the build thusly.

Reply:
```suggestion
    WORK_DIR="${{ runner.temp }}"
```